src/applications/appeals/995/subtask/pages/start.jsx

```diff
@@ -113,6 +117,7 @@ const BenefitType = ({ data = {}, error, setPageData }) => {
             value={value}
             label={label}
             checked={value === data.benefitType}
+            uswds
```

Comment:
Oh same question here. Is this group `<VaRadioOption>` & `<VaRadio>` the same as `<va-radio-option>` & `<va-radio>` respectively? I recall something about the `react-bindings`

Reply:
Yep, I can't remember why this needs `VaRadioOption`, but because the `VaRadio` is using `onVaValueChange` we need to use the React binding